src/lib/Scenes/City/cities.ts

```diff
@@ -3,48 +3,48 @@ export const cityList = [
     name: "New York",
     slug: "new-york-ny-usa",
     epicenter: {
-      lat: 40.7128,
-      lng: -74.006,
+      lat: 40.71,
+      lng: -74.01,
     },
   },
   {
     name: "Los Angeles",
     slug: "los-angeles-ca-usa",
     epicenter: {
-      lat: 34.0522,
-      lng: 118.2437,
+      lat: 34.05,
+      lng: -118.24,
```

Comment:
This is the fix for the Los Angeles issue — sign needed to be flipped for decimal west longitude.

(The reduction in precision here and throughout is appropriate for city-scale mapping.)

Reply:
Good catch! Is there a name for this specific kind of bug? Where a latitude or longitude is accidentally inverted?

Reply:
Not aware of any special name for this, just a good ol fashioned sign error 😄